src/app/_services/error.service.ts

```diff
@@ -56,6 +56,7 @@ export class ErrorService {
           }),
           take(this.numberOfRetries)
         ),
+        // eslint-disable-next-line rxjs/throw-error
```

Comment:
I know we discussed this, but only within the context of your mocked service. This is actual production code. 
Is there no way to do this more elegantly? Why not pass an instance of `HttpErrorResponse`?